Sources/Sentry/SentryFileManager.m

```diff
@@ -147,6 +150,41 @@ - (NSString *)storeEnvelope:(SentryEnvelope *)envelope {
     }
 }
 
+- (void)storeCurrentSession:(SentrySession *)session {
+    NSData *sessionData = [SentrySerialization dataWithSession:session options:0 error:nil];
+    [SentryLog logWithMessage:[NSString stringWithFormat:@"Writing to current session: %@", self.currentSessionFilePath] andLevel:kSentryLogLevelDebug];
+    @synchronized (self.currentSessionFilePath) {
+        [sessionData writeToFile:self.currentSessionFilePath options:NSDataWritingAtomic error:nil];
```

Comment:
Aren't we interested in a possible error? Is it fine to fail silently?

Reply:
We'll need to go after these in a separate PR. There are quite a few already